src/hotspot/share/logging/logTag.hpp

```diff
@@ -212,6 +212,7 @@ class outputStream;
   LOG_TAG(valuebasedclasses) \
   LOG_TAG(verification) \
   LOG_TAG(verify) \
+  LOG_TAG(vmlocks) \
   LOG_TAG(vmmutex) \
```

Comment:
Why not just use the existing `vmmutex` tag?

Reply:
Fixed. Now the logging is enabled via `-Xlog:perf+vmmutex`.